### PR TITLE
Don't trigger sample tests run after a change in non yml related files in a package

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
In Progress

## Description

- [x] Don't trigger sample tests run after a change in non yml related files
- [x] Run configure_tests test on Circle

### CODE REVIEW

**Comment by glicht** [comment]
@ronykoz I committed a unit test I wrote before the fix. It is passing now. Also created the dev-task: https://github.com/demisto/etc/issues/16474 to fix the other failing tests.

**Comment by ronykoz** [comment]
@glicht should we spend time adding more tests here or should I merge it this way?

File: .circleci/config.yml [hunk 1]
```diff
@@ -28,15 +28,16 @@ jobs:
...
       - run:
           name: Infrastucture testing
           command: |
             pytest ./Tests/scripts/hook_validations/tests/ -v
             pytest ./Tests/scripts/infrastructure_tests/ -v
-      - run:
-          name: Create ID Set
-          command: |
-            python ./Tests/scripts/update_id_set.py -r
+            python ./Tests/scripts/test_configure_tests.py
```
Comment (glicht) [inline .circleci/config.yml]:
```suggestion
            pytest ./Tests/scripts/test_configure_tests.py -v
```
Let's run the tests with pytest. The output is cleaner and easier to understand when there is a failure.

**Comment by glicht** [comment]
Looks good. Don't think we need more tests.

**Comment by ronykoz** [comment]
@glicht good to go

**Comment by glicht** [comment]
feel free to merge

**Pull request merged**